src/applications/yellow-ribbon/components/FindYellowRibbonPage/index.js

```diff
@@ -18,21 +19,19 @@ export const FindYellowRibbonPage = () => (
           you already have Post-9/11 GI Bill benefits, this program may help you
           pay for tuition for a degree or training program at a college or other
           institution of higher learning. The amount of money you get varies by
-          school, degree type, and the academic program you&apos;re enrolled in.
+          school, degree type, and the academic program you’re enrolled in.
```

Comment:
Are we moving away from using html codes?

Reply:
I guess so, that was the ESlint fix from that auto fixes apostrophes